src/index.d.ts

```diff
@@ -873,7 +873,7 @@ export interface InputProps extends TextInputProperties {
   /**
    * 	Adds label (optional)
    */
-  label?: string;
+  label?: string  | React.ComponentElement;
```

Comment:
Use this React.ReactElement instead. See https://github.com/react-native-training/react-native-elements/blob/e7ce592fa83a614b7cd1cce6f8a4e0795136775c/src/index.d.ts#L1100